### ISSUE TITLE
ACC-264.1 (Level A) _ Modify Modules _ _ Table title not announced 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3140]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	-
Affected Check Point:	ACC-264.1
Short text:	ACC-264.1 (Level A) _ Modify Modules _ _ Table title not announced 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Now press ‘T’ key to know the details of tables within the page.

3. Issue Type: Manual
Observed Behavior:
On pressing ‘T’ key , incorrect information ‘There are no tables in this page’ is  announced by the screen reader whereas the user can  see a visible table ‘Modules’ within the page.
Screenshot:
 

Expected Behavior:
On pressing ‘T’ key , correct information about the table ‘Modules’ must be announced by the screen reader along with other necessary information.

________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-264.1 (Level A) _ Modify Modules _ _ Table title not announced.docx](https://github.com/user-attachments/files/16659883/ACC-264.1.Level.A._.Modify.Modules._._.Table.title.not.announced.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16659884/Issue.processing-09.2021.002.docx)

**Comment by OliwiaGowor** [comment]
It's connected to this [issue](https://github.com/SAP/ui5-webcomponents/issues/10747) - we will have to wait for UI5 React 2.7 release.
